examples/integrations/cdi/datasource-hikaricp/README.adoc

```diff
@@ -0,0 +1,108 @@
+= Hikari Connection Pool Integration Example
+
+== Overview
+
+This example shows a trivial Helidon MicroProfile application that
+uses the Hikari connection pool CDI integration, shows how to package
+it into a Docker image, and shows how to run it in a Docker container.
+It also shows how to run the Oracle database in a Docker container and
+connect to it using the application's container.
+
+== Notes
+
+To create a user-defined Docker network:
+
+```sh
+$ docker network create helidon-examples
+```
+
+To log in to the Oracle Container Registry (which you will need to do
+in order to pull Oracle database images from it):
+
+```sh
+$ docker login -u your.name@yourdomain.com -p yourpasswordhere container-registry.oracle.com
```

Comment:
What kind of account is that ? Should we document the steps on how to setup this account ?

There is also `store/oracle/database-enterprise:12.2.0.1-slim` available from the docker store with a few manual steps (mostly providing user information).

Reply:
https://blogs.oracle.com/weblogicserver/the-oracle-container-registry-has-gone-live

Reply:
We should probably include some of these instructions from the blog post in this file.
Referencing a weblogic blog post does not seem right :D

Reply:
Do we need to reference anything at all, actually?  I'm not sure the blog post or the registry website offer much above the recipe above.  WDYT?

Reply:
The current recipe shows "-u your.name@yourdomain.com" but does not explain how to get that value. Going to container-registry.oracle.com shows a login button but that's it.

Reply:
Fixed

Reply:
I found this: https://docs.oracle.com/cd/E37670_01/E75728/html/oracle-registry-server.html